server/routes/api/application.js

```diff
@@ -1,36 +1,83 @@
 var router = require('express').Router(),
     Responses = require('../../responses/api'),
-    Application = require('../../db/model/Application.js');
+    User = require('../../db/model/User.js'),
+    Application = require('../../db/model/Application.js'),
+    authMiddleware = require('../../middleware/auth.js'),
+    config = require('../../../config/default.js'),
+    uploadHelper = require('../../interactors/multer-s3.js')(
+        config.AWS_BUCKET_NAME
+    );
 
-router.post('/', function(req, res) {
-    Application.find().byToken(req.authToken).then(query => {
-        query
+router.post(
+    '/',
+    authMiddleware('any', 'api', true),
+    uploadHelper.fields([{ name: 'resume' }]),
```

Comment:
My goal was to have it either referencing the existing resume or uploading a new one if it isn't there